src/img/chevronRight.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 5l7 7-7 7" />
+</svg>
```

Comment:
I'm not sure we need these imported as images. I feel like it's safe to delete this and the four other SVG image files.

Reply:
It was mostly an organization thing that I had going on, which morphed from me initially wondering "um... do I have to have img files to use these icons???" for a moment there in the beginning. They all would have benefitted from some comments! Intent was to clean those up at the end, but again, I should have added that comment. Removal now is a great call.